Tests/SwiftLintFrameworkTests/RulesTests.swift

```diff
@@ -133,6 +135,10 @@ class RulesTests: XCTestCase {
         verifyRule(FunctionParameterCountRule.description)
     }
 
+    func testNoGroupingExtension() {
```

Comment:
just one more thing: we try to keep this list sorted. Can you please move it (and run `make sourcery` again)? 😅

Reply:
Of course! Sorry about that.